apps/webstore/src/app/shared/components/modals/add-to-basket-modal/add-to-basket-modal.component.ts

```diff
@@ -209,6 +213,17 @@ export class AddToBasketModalComponent implements OnInit, OnChanges {
         }
         break;
       }
+      case DesignTypeEnum.petSign: {
```

Comment:
Same here as before. Since its the same sizes as quotable, cant it just be added to the same case flow? :)